testing/generate.js

```diff
@@ -213,7 +213,7 @@ function main() {
   }
   
   const sourceControlPath = path.join(basePath, '.retool', 'protected-apps.yaml');
-  if (!fs.existsSync(sourceControlPath)) {
+  if (usingSourceControl && !fs.existsSync(sourceControlPath)) {
     console.log('Source Control repository is missing ./retool/protected-apps.yaml');
```

Comment:
i believe this should be /.retool/protected-apps.yaml?

Reply:
https://github.com/tryretool/retool-onpremise/pull/101